File: PeopleCounter/app/src/main/java/com/example/sujan/peoplecounter/MainActivity.java
package com.example.sujan.peoplecounter;

import android.os.Bundle;
import android.support.wearable.activity.WearableActivity;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends WearableActivity {

    private TextView mTextView;

    int count = 0;

    public void increment(View view) {
        count++;
        mTextView.setText(Integer.toString(count));
    }

    public void reset(View view) {
        count = 0;
        mTextView.setText(Integer.toString(count));
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mTextView = (TextView) findViewById(R.id.text);

        // Enables Always-on
        setAmbientEnabled();
    }
}

File: KotlinFun/app/src/main/java/com/example/sujan/kotlinfun/MainActivity.kt
package com.example.sujan.kotlinfun

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var count = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var textView = findViewById<TextView>(R.id.textView)
        textView.text = "0"
    }

    fun plusOne(view : View) {
        count += 1
        textView.text = count.toString()
    }

    fun reset(view : View) {
        count = 0
        textView.text = count.toString()
    }
}
